### ISSUE TITLE
Unable to activate Split Button with triple tap in ToggleSplitButton pane.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#166]
**Describe the bug**

**Pre-requisite:** 
Turn on Narrator.
Set Narrator in Item Mode.
Touch supported monitor required.

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery application.
2. Navigate to "All Controls" button and activate it.
3. Navigate to "ToggleSplitButton" and activate it.
4. Triple tap on Split button to activate it and observe the Text Box.

**Expected behavior**

When user triple tap on split button, then either Roman numeral or Bullet list index, whichever is selected should appear in the text box e.g  "i)" or a Bullet.

**Actual behavior**

When user triple tap on split button then state of button changes i.e On/Off, but selected index value type does not appear in the text box.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/63800616-63336000-c8c3-11e9-8557-19a02e58caac.png)

**Version Info**

Gallery version (found on Settings page): 

App Version: 1.2.8.0

Windows 10 version:
- [x] Insider Build (xxxxx)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/22658341)
